### PR TITLE
Op (Save/LoadCombine) error message enhancement

### PR DESCRIPTION
**Pull request by chenwhql**
## Save/LoadCombine Op报错信息增强
- 两个Op关联较强，一起修改
- 仅修改了C++端的报错检查

### CODE REVIEW

File: paddle/fluid/operators/load_combine_op.h [hunk 1]
```diff
@@ -35,21 +35,27 @@ class LoadCombineOpKernel : public framework::OpKernel<T> {
...
     if (!model_from_memory) {
       std::ifstream fin(filename, std::ios::binary);
-      PADDLE_ENFORCE(static_cast<bool>(fin),
-                     "OP(LoadCombine) fail to open file %s, please check "
-                     "whether the model file is complete or damaged.",
-                     filename);
+      PADDLE_ENFORCE_EQ(
+          static_cast<bool>(fin), true,
+          platform::errors::Unavailable(
+              "LoadCombine operator fail to open file %s, please check "
```
Comment (sneaxiy) [inline paddle/fluid/operators/load_combine_op.h]: `fail` -> `fails`?

File: paddle/fluid/operators/load_combine_op.h [hunk 2]
```diff
@@ -35,21 +35,27 @@ class LoadCombineOpKernel : public framework::OpKernel<T> {
...
       LoadParamsFromBuffer(ctx, place, &fin, load_as_fp16, out_var_names);
     } else {
-      PADDLE_ENFORCE(!filename.empty(),
-                     "OP(LoadCombine) fail to open file %s, please check "
-                     "whether the model file is complete or damaged.",
-                     filename);
+      PADDLE_ENFORCE_NE(
+          filename.empty(), true,
+          platform::errors::Unavailable(
+              "LoadCombine operator fail to open file %s, please check "
```
Comment (sneaxiy) [inline paddle/fluid/operators/load_combine_op.h]: `fail` -> `fails`?

File: paddle/fluid/operators/save_combine_op.h [hunk 3]
```diff
@@ -41,31 +41,40 @@ class SaveCombineOpKernel : public framework::OpKernel<T> {
 
     bool is_present = FileExists(filename);
     if (is_present && !overwrite) {
-      PADDLE_THROW("%s exists!, cannot save_combine to it when overwrite=false",
-                   filename, overwrite);
+      PADDLE_THROW(platform::errors::PreconditionNotMet(
+          "%s exists!, cannot save_combine to it when overwrite is set to "
```
Comment (sneaxiy) [inline paddle/fluid/operators/save_combine_op.h]: Remove `,` and capitalize the 1st letter of `cannot`.

File: paddle/fluid/operators/load_combine_op.h (hunk 1 above)
Comment (chenwhql) [reply paddle/fluid/operators/load_combine_op.h]: done, thx

File: paddle/fluid/operators/load_combine_op.h (hunk 2 above)
Comment (chenwhql) [reply paddle/fluid/operators/load_combine_op.h]: done, thx

File: paddle/fluid/operators/save_combine_op.h (hunk 3 above)
Comment (chenwhql) [reply paddle/fluid/operators/save_combine_op.h]: done, thx

**Review by phlrain — APPROVED** [review]

**Pull request merged**